static/app/views/replays/detail/breadcrumbs/index.tsx

```diff
@@ -79,7 +79,7 @@ function Breadcrumbs({}: Props) {
         // XXX: Kind of hacky, but mouseLeave does not fire if you move from a
         // crumb to a tooltip
         clearAllHighlights();
-        highlight({nodeId: item.data.nodeId});
+        highlight({nodeId: item.data.nodeId, annotation: item.data.label ?? ''});
```

Comment:
I think this would satisfy the types (well maybe not if it's null):

```suggestion
        highlight({nodeId: item.data.nodeId, annotation: item.data.label});
```

I don't think there's going to be a difference downstream between `undefined` and `''` though. Downstream is here: https://github.com/getsentry/sentry/blob/master/static/app/views/replays/detail/breadcrumbs/breadcrumbItem.tsx#L63

Reply:
Ah downstream is actually this, which defaults to empty string: https://github.com/getsentry/sentry/blob/e2e3bac89ea8d8d0c9f6e90a485364f5a1fd6ef3/static/app/utils/replays/highlightNode.tsx#L58